CDP4Common/AutoGenPoco/ReferenceDataLibrary.cs

```diff
@@ -257,6 +257,36 @@ public override IEnumerable<IEnumerable> ContainerLists
             }
         }
 
+        /// <summary>
+        /// Queries the referenced <see cref="Thing"/>s of the current <see cref="ReferenceDataLibrary"/>
+        /// </summary>
+        /// <remarks>
+        /// this does not include the contained <see cref="Thing"/>s, the contained <see cref="Thing"/>s
+        /// are exposed via the <see cref="ContainerLists"/> method
+        /// </remarks>
+        /// <returns>
+        /// An <see cref="IEnumerable{Thing}"/>
+        /// </returns>
+        public override IEnumerable<Thing> QueryReferencedThings()
+        {
+            foreach (var thing in base.QueryReferencedThings())
+            {
+                yield return thing;
+            }
+
+            foreach (var thing in this.BaseQuantityKind.Select(x => x))

```

Comment:
I noticed that for some things (most don't have this) that are iterated there's an additional `.Select(x =>x)`. Does this serve any purpose? It seems to me like it's adding [execution] complexity for nothing.

Reply:
It is required to handle ordereditemlist